sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java

```diff
@@ -267,6 +271,7 @@ public String getMessageId() {
      * @return The updated {@link ServiceBusMessage}.
      */
```

Comment:
Since you are adding validation, I would add a `@throws` to say what runtime exceptions are thrown.

Reply:
Added.